### DISCUSSION TITLE
Graphql Client Subscription Example

### DISCUSSION THREAD

**Discussion by tawabocah** [Netflix/dgs-framework#466] [Q&A]
Hi, any sample on how to write client for subscription endpoint + graphql query?
I find the netflix doc, but can't find any.

**Comment by timpamungkas** [comment]
Not sure if DGS client can do this. Try using [apollo graphql](https://www.apollographql.com/docs/android/tutorial/11-subscriptions/) client instead

**Comment by paulbakker** [comment]
If you're asking about how to do this in a web UI, we have a simple example based on Apollo here: https://github.com/Netflix/dgs-framework/blob/master/graphql-dgs-example-shared/ui-example/src/index.tsx#L39

The Apollo docs are great, so that's a great place to learn more as well.
